packages/components/Avatar/src/useAvatar.ts

```diff
@@ -161,6 +167,36 @@ export const removeTitlesFromName = (words: string[]): string[] => {
   return words.filter((word) => !titles.has(word));
 };
 
+/**
+ * A function which verifies whether words array contains multi-word title
+ *
+ * @param name
+ * @returns name without multi-word titles
+ */
+const removeMultiWordTitles = (name: string, idx: number = 0): string => {
+  if (idx >= multiWordTitles.length) {
+    return name;
+  }
+
+  const title = multiWordTitles[idx];
+
+  if (name.indexOf(title) !== -1) {
+    return name.replace(title, '');
+  }
+
+  return removeMultiWordTitles(name, idx + 1);
```

Comment:
nit: I prefer not using recursion if the code won't be much longer using a loop instead, since recursion is more expensive/slow performantly and I also find it harder to reason/read. But that's a personal preference, you can keep the code as it is if you'd prefer.

Reply:
In this case I used recursion because the array contains only 40 items, I thought it won't hurt performance. But I'll change to the loop